packages/components/src/Page/Page.tsx

```diff
@@ -107,6 +114,15 @@ export function Page({
     <div className={pageStyles}>
       <Content>
         <Content>
+          {breadcrumb && (
+            <Button
+              icon="backArrow"
+              url="../"
```

Comment:
For added flexibility, would we be able to have 2 additional properties of a breadcrumb?


```typescript
interface breadcrumb {
  label: string;
  /* Default = "../" */
  url?: string;
  onClick?(): void;
}
```

If the `onClick` property is passed, the url is ignored